src/syscall/syscall_nintendoswitch.go

```diff
@@ -0,0 +1,41 @@
+// +build nintendoswitch
+
+package syscall
+
+// A Signal is a number describing a process signal.
+// It implements the os.Signal interface.
+
```

Comment:
Nit: this empty line removes the doc string from the `Signal` type.

Reply:
@racerxdl this as well.

Reply:
Done!